File: examples/node/paymentCreate.ts
import { post, SHOP, TERMINAL } from "./common.js";
// Static QR terminal example (change productCode to BSC_PAYMENT_CODE for B-scan-C and add paymentCode)
const path = "/v1/acquiring/payment/create";
const body = {
  productCode: "CSB_DIRECTPAY_OFFLINE_TERMINALQR",
  shopId: SHOP,
  terminalId: TERMINAL,
  order: { orderTitle: "Static QR", merchantTransId: "m-"+Date.now(), orderAmount: { value: "1200", currency: "MXN" } }
};
post(path, body).then(console.log).catch(e => console.error(e.response?.data || e));

File: examples/python/payment_create.py
import os, time
from common import post
body = {
  "productCode":"CSB_DIRECTPAY_OFFLINE_TERMINALQR",
  "shopId": os.environ["TOKAPAY_SHOP_ID"],
  "terminalId": os.environ["TOKAPAY_TERMINAL_ID"],
  "order":{"orderTitle":"Static QR","merchantTransId":"m-"+str(int(time.time()*1000)),"orderAmount":{"value":"1200","currency":"MXN"}}
}
print(post("/v1/acquiring/payment/create", body))
